hikari/api/rest.py

```diff
@@ -780,6 +817,43 @@ async def create_message(
             once before being able to use this endpoint for a bot.
         """  # noqa: E501 - Line too long
 
+    @abc.abstractmethod
+    async def crosspost_message(
+        self,
+        channel: snowflakes.SnowflakeishOr[channels.GuildNewsChannel],
+        message: snowflakes.SnowflakeishOr[messages_.Message],
+    ) -> messages_.Message:
+        """Crosspost a message in a News Channel to following channels.
+
+        Parameters
+        ----------
+        channel : hikari.snowflakes.SnowflakeishOr[hikari.channels.GuildNewsChannel]
+            The object or ID of the news channel to crosspost a message in.
```

Comment:
```suggestion
            The object or ID of the text announcement channel to crosspost a
            message in.
```

Reply:
Seems a bit odd to put it like that when we call them news channels internally